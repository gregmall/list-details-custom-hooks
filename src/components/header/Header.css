Header{
  background-color:rgb(41, 201, 212);
  height: 75px;
  width: 100%;
  margin-bottom: 10px;
  display: flex;
  justify-content: space-between;;
  flex-direction: row;

}
