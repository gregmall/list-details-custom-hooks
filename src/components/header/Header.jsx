
import React from 'react';
import styles from './Header.css'

const Header = () => {
  return (
    <header className={styles.Header}>
  
      <div>
          <img src="src/components/header/Webp.net-resizeimage.png" />
      </div>
      <div>
          <a href="/">HOME</a>
      </div>
      <div>
          <h1>
            RICK AND MORTY
          </h1>
      </div>
    </header>
  );
};
export default Header;
