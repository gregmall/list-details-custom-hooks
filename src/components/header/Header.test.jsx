import React from 'react'
import { render, cleanup } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Header from './Header';

describe('Header component', () => {
  afterEach(() => cleanup());
  it('renders the Header to the page', () => {
    const { asFragment } = render(
      <MemoryRouter>
        <Header />
      </MemoryRouter>
    );
      expect(asFragment()).toMatchSnapshot();
  });
});
