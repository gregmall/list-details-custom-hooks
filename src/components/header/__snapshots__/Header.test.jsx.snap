// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders the Header to the page 1`] = `
<DocumentFragment>
  <header
    class="Header"
  >
    <div>
      <img
        src="src/components/header/Webp.net-resizeimage.png"
      />
    </div>
    <div>
      <a
        href="/"
      >
        HOME
      </a>
    </div>
    <div>
      <h1>
        RICK AND MORTY
      </h1>
    </div>
  </header>
</DocumentFragment>
`;
